pkg/server/server_authz_test.go

```diff
@@ -700,6 +707,80 @@ func TestCheckAuthz(t *testing.T) {
 	})
 }
 
+func TestGetAccessibleStores(t *testing.T) {
+	t.Cleanup(func() {
+		goleak.VerifyNone(t)
+	})
+	ds := memory.New()
+	t.Cleanup(ds.Close)
+
+	t.Run("check_no_authz", func(t *testing.T) {
+		openfga := MustNewServerWithOpts(
+			WithDatastore(ds),
+		)
+		t.Cleanup(openfga.Close)
+
+		_, err := openfga.getAccessibleStores(context.Background())
+		require.NoError(t, err)
+	})
+
+	t.Run("check_with_authz", func(t *testing.T) {
+		openfga := MustNewServerWithOpts(
+			WithDatastore(ds),
+		)
+		t.Cleanup(openfga.Close)
+
+		clientID := "validclientid"
+		settings := newSetupAuthzModelAndTuples(t, openfga, clientID)
+
+		openfga.authorizer = authz.NewAuthorizer(&authz.Config{StoreID: settings.rootData.id, ModelID: settings.rootData.modelID}, openfga, openfga.logger)
+
+		t.Run("with_SkipAuthzCheckFromContext_set", func(t *testing.T) {
+			ctx := authclaims.ContextWithSkipAuthzCheck(context.Background(), true)
+
+			_, err := openfga.getAccessibleStores(ctx)
+			require.NoError(t, err)
+		})
+
+		t.Run("error_with_no_client_id_found", func(t *testing.T) {
+			_, err := openfga.getAccessibleStores(context.Background())
+			require.Error(t, err)
+			require.Equal(t, "rpc error: code = InvalidArgument desc = client ID not found in context", err.Error())
```

Comment:
Will it be better to use `require.ErrorIs(t, fmt.Errorf("rpc error: code = InvalidArgument desc = client ID not found in context", err)`?